src/Kunstmaan/FormBundle/DependencyInjection/Configuration.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Kunstmaan\FormBundle\DependencyInjection;
 
+use Kunstmaan\MediaBundle\Utils\SymfonyVersion;
```

Comment:
The formbundle has no dependency on the media bundle, so technically the class might not be available. So for the time being (as this is a sf 3/4 bc layer) we can inline the check here?

Reply:
Done